src/components/LoginPage.tsx

```diff
@@ -0,0 +1,40 @@
+import { useFirebase } from 'react-redux-firebase'
+import { useNavigate } from 'react-router-dom'
+import { AppBar, Button, Toolbar, Typography } from '@mui/material'
+
+export const LoginPage = () => {
+  const firebase = useFirebase()
+  const navigate = useNavigate()
+
+  const signInWithGoogle = () => {
+    firebase
+      .login({
+        provider: 'google',
+        type: 'popup',
+      })
+      .then(() => {
+        navigate('/dashboard')
+      })
+  }
```

Comment:
Tests would be appreciated 😈  You may test here following elements:
1. `firebase.login` has been called
2. navigate user to `/dashboard` after that